cmake/kokkos_tribits.cmake

```diff
@@ -207,6 +207,11 @@ MACRO(KOKKOS_SETUP_BUILD_ENVIRONMENT)
     IF (NOT KOKKOS_HAS_TRILINOS)
       SET(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${Kokkos_SOURCE_DIR}/cmake/Modules/")
       INCLUDE(${KOKKOS_SRC_PATH}/cmake/kokkos_tpls.cmake)
+    ELSE()
+      IF (Kokkos_ENABLE_LIBDL)
+        KOKKOS_IMPORT_TPL(LIBDL)
+      ENDIF()
+    ENDIF()
```

Comment:
```suggestion
```

Reply:
Force-pushed fix, must've typoed when cleaning up, thanks for catch